CodePush.js

```diff
@@ -401,6 +401,46 @@ async function syncInternal(options = {}, syncStatusChangeCallback, downloadProg
 
 let CodePush;
 
+function codePushify(options = {}) {
+  return (RootComponent) => {
+    let React;
+    let ReactNative = require("react-native");
+
+    try { React = require("react"); } catch (e) { }
+    if (!React) {
+      try { React = ReactNative.React; } catch (e) { }
+      if (!React) {
+        throw new Error("Unable to find the 'React' module.");
+      }
+      if (!React.Component) {
```

Comment:
Should this `if` statement be outside of the `if (!React)` block, so that we can check it even if `React` was successfully found the first time?

Also, I'm not sure we need (2) - do we expect this to happen often?